CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,37 @@
+#Code of Conduct
```

Comment:
Sorry, I've missed this in the last review but can you add a space between the "#" and the title? Otherwise is not displayed correctly.